tests/acceptance/features/html-output.feature

```diff
@@ -0,0 +1,46 @@
+Feature: HTML output
+
+  As a package maintainer
+  I want to execute the HTML formatter and verify its output
+  So that I can ensure integration and basic functionality.
+
+  Scenario:
```

Comment:
As starting point OK.
BETTER (alternatives): 
* Scenario: Use formatter=html 
* Scenario: Use HTML Formatter

BUT: Scenario without name/title is a SMELL because:
* it lacks the readable/understandable intention why it exists and what is tested here
* Some formatter (like: progress3) outputs will be less readable/understandable

Reply:
I overlooked that completely, this was not intended. Sorry! I'll get that fixed.